viewing_party/party.py

```diff
@@ -1,23 +1,163 @@
 # ------------- WAVE 1 --------------------
+from statistics import mean, mode
 
 def create_movie(title, genre, rating):
-    pass
 
-# -----------------------------------------
-# ------------- WAVE 2 --------------------
-# -----------------------------------------
+    if title and genre and rating:
+    
+        movie = {"title": title,
+    "genre": genre,
+    "rating": rating,
+    }
+
+        return movie
+ 
+def add_to_watched(user_data, movie):
+
+    movies_watched = movie
+    user_data["watched"].append(movies_watched)
+
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+
+    movie_watchlist = movie
+    user_data["watchlist"].append(movie_watchlist)
+
+    return user_data
+
+
+def watch_movie(user_data, title):
+    watchlist = user_data["watchlist"]
+    
+    
+    for watch in range(len(watchlist)):
+        if watchlist[watch]["title"] == title: 

```

Comment:
Instead of using `range` to loop over the length of the watchlist and accessing `watchlist[watch]["title"]` you could loop over `watchlist` and for each watch you could access and compare the title. Something like this:
```
for movie in user_data["watchlist"]:
        if movie["title"] == title:
```

Or this:
```
watchlist = user_data["watchlist"]

for watch in watchlist:
        if watch["title"] == title:
```

You could also call the previous function to add to the watchlist instead of appending:

```
if movie["title"] == title:
            user_data["watchlist"].remove(movie)
            add_to_watched(user_data, movie)
```